src/networks.js

```diff
@@ -0,0 +1,55 @@
+import env from './environment'
+
+const DEFAULT_IPFS_GATEWAY = 'https://ipfs.eth.aragon.network/ipfs'
+const DEFAULT_ORG_LOCATION = 'agreement3.aragonid.eth'
```

Comment:
I think we might as well specify these two settings to be per-network (especially the organization location).

Reply:
Sure!